bin/generate

```diff
@@ -54,17 +54,17 @@ rm -rfv test/
 go mod tidy
 
 # Work around openapi-generator (or at least the go generator) not understanding x-nullable enums
-sed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token.go
-sed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token.go
-sed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request.go
-sed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request.go
-sed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request_patch.go
-sed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request_patch.go
-sed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh.go
-sed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh.go
-sed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh_request.go
-sed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh_request.go
-sed -i 's/ACTION/string/g' model_package_tag_request.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_request_patch.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_request_patch.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh.go
+gsed -i 's/LIMIT_BANDWIDTH_UNIT/string/g' model_repository_token_refresh_request.go
+gsed -i 's/SCHEDULED_RESET_PERIOD/string/g' model_repository_token_refresh_request.go
+gsed -i 's/ACTION/string/g' model_package_tag_request.go
```

Comment:
Just curious, what's the advantage of GNU `sed` over the former?

Reply:
no advantage, sed doesnt work with the mac version.